src/identifier.h

```diff
@@ -21,12 +21,18 @@
 
 class Identifier : public RootObject
 {
+private:
```

Comment:
GDC has a few dozen examples of using `ident->string` directly in the code generator, being told that it is private is more meaningful than being told it doesn't exist.  Having this here also means I can just use this header "as-is" as I rebase the C++ frontend onto the stable branch.